src/tests/test_dynamic_api/test_router.py

```diff
@@ -92,3 +92,14 @@ def test_router_excludes_datasets_combined_with_list(
         assert reverse("dynamic_api:bommen-bommen-list")
     with pytest.raises(NoReverseMatch):
         assert reverse("dynamic_api:meldingen-statistieken-list")
+
+
+@pytest.mark.django_db
+def test_router_excludes_non_default_dataset_versions(settings, router, bommen_v2_dataset):
+    router.reload()
```

Comment:
after my faster tests merge:

```suggestion
def test_router_excludes_non_default_dataset_versions(settings, bommen_v2_dataset, filled_router):
```

Reply:
But it is not merged yet :)

Reply:
En zo heb ik het gedaan:
```
def test_router_excludes_non_default_dataset_versions(settings, bommen_v2_dataset, filled_router):
```

>>

```
E                   RuntimeError: The 'filled_router' or 'dynamic_models' fixture was requested, but no dataset fixtures were defined for this test
```
:(

Reply:
Added comment on why `filled_router` can't be used in this case.